src/components/MemberManager/index.js

```diff
@@ -0,0 +1,183 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {ScrollView, View} from 'react-native';
+import {List, ListItem, Divider} from 'react-native-elements';
+import {Button, LoadingIndicator, Title} from '@indec/react-native-commons';
+import {getFontAwesome} from '@indec/react-native-commons/util';
+import {findIndex, isNil, max} from 'lodash';
+
+import NavigationButtons from '../NavigationButtons';
+import Form from '../Form';
+import {Member} from '../../model';
+import {questionPropTypes, cleanChildrenQuestions} from '../../util';
+import {requestMembers, requestSaveMembers} from '../../actions/survey';
+import matchParamsIdPropTypes from '../../util/matchParamsIdPropTypes';
+import styles from './styles';
+
+const isSelected = (selected, member) => !isNil(selected) && member.order === selected.order;
+
+class MemberManager extends Component {
+    static propTypes = {
+        requestMembers: PropTypes.func.isRequired,
+        requestSaveMembers: PropTypes.func.isRequired,
+        match: matchParamsIdPropTypes.isRequired,
+        members: PropTypes.arrayOf(
+            PropTypes.instanceOf(Member)
+        ),
+        rows: questionPropTypes.isRequired,
+        rowsForHomeBoss: questionPropTypes.isRequired,
+        onPrevious: PropTypes.func.isRequired,
+        onSubmit: PropTypes.func.isRequired
+    };
+
+    static defaultProps = {
+        members: []
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selected: null
+        };
+    }
+
+    componentWillMount() {
+        const {id, dwelling, household} = this.props.match.params;
+        this.props.requestMembers(id, dwelling, household);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.members) {
+            this.state.members = nextProps.members;
+        }
+    }
+
+    getRows({homeBoss}) {
+        return homeBoss ? this.props.rowsForHomeBoss : this.props.rows;
+    }
+
+    handleChange(answer, rows) {
+        this.setState(state => {
+            const expandedData = Object.assign(state.selected.expandedData, answer);
+            return ({
+                selected: cleanChildrenQuestions(
+                    rows,
+                    Object.assign(state.selected, expandedData)
+                )
+            });
+        });
+    }
+
+    add() {
+        const {members} = this.state;
```

Comment:
en lugar de tomar esto del `this.state` mete todo dentro del `arrow function` del `this.setState(state => {...});`.

Reply:
Listo